source/uwp/UWPTestLibrary/TestResultViewModel.cs

```diff
@@ -317,41 +446,70 @@ private static string GetStoredSourceFileName(string name, string hash)
         }
     }
 
-    public enum TestStatus
+    public class TestStatus
     {
-        /// <summary>
-        /// Visual identically matched
-        /// </summary>
-        Passed,
-
-        /// <summary>
-        /// Visual did match, but source files (payload and host config) changed, so changes are possibly expected
-        /// </summary>
-        PassedButSourceWasChanged,
-
-        /// <summary>
-        /// Visual did NOT match, and source files (payload and host config) were identical, Json round tripping matches
-        /// </summary>
-        Failed,
-
-        /// <summary>
-        /// Visual did NOT match, and source files (payload and host config) were identical, Json round tripping doesn't match
-        /// </summary>
-        ImageAndJsonFailed,
-
-        /// <summary>
-        /// Visual identically matched, Roundtripped json did not match, and source files (payload and host config) were identical
-        /// </summary>
-        JsonFailed,
-
-        /// <summary>
-        /// Visual did NOT match, but source files (payload and host config) changed, so changes are possibly expected
-        /// </summary>
-        FailedButSourceWasChanged,
-
-        /// <summary>
-        /// New test, no existing Expected
-        /// </summary>
-        New
+        // The error results match (either both pass or both fail with the same result)
+        public bool MatchedViaError { get; set; } = false;
```

Comment:
NIT: Use the 3 slashes comments like `/// <summary>`, those will be picked up by IntelliSense so that when you're accessing these properties elsewhere, their summaries appear :)